doc/modules/compose.rst

```diff
@@ -443,8 +446,9 @@ By default, the remaining rating columns are ignored (``remainder='drop'``)::
 
 In the above example, the
 :class:`~sklearn.feature_extraction.text.CountVectorizer` expects a 1D array as
-input and therefore the columns were specified as a string (``'city'``).
-However, other transformers generally expect 2D data, and in that case you need
+input and therefore the columns were specified as a string (``'title'``).
+However, :class:`preprocessing.OneHotEncoder <sklearn.preprocessing.OneHotEncoder>`
```

Comment:
You don't need the stuff in the angle brackets

Reply:
It was there originally. @jorisvandenbossche reading those lines yesterday I did not why you used this syntax? Any reasons?

Reply:
The reason it is written like this is because the "currentmodule" of sphinx is `sklearn.pipeline`, so I think we need to use the full path to refer to `sklearn.preprocessing.OneHotEncoder`. What this then does is shorten that in the *display* to `preprocessing.OneHotEncoder` (the functionality to link to its docstring page of course stays the same).

Whether this complexity is worth it, that's another question :)

But to be consistent with how the rest of the text is, it can be replaced here with `` :class:`~sklearn.preprocessing.OneHotEncode` `` (which will just display `OneHotEncoder`)